components/test_coprocessor/Cargo.toml

```diff
@@ -5,10 +5,12 @@ edition = "2018"
 publish = false
 
 [dependencies]
-tikv_util = { path = "../tikv_util" }
 tikv = { path = "../../", default-features = false }
 tipb = { git = "https://github.com/pingcap/tipb.git" }
 kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 protobuf = "2"
-test_storage = { path = "../test_storage" }
 futures = "0.1"
+
```

Comment:
Does this blank line have some special intention?

Reply:
For separating workspace dependencies and 3rd-party dependencies.